doc/doxyfile.in

```diff
@@ -44,14 +44,14 @@ PROJECT_NUMBER         = @VERSION_MAJOR@.@VERSION_MINOR@.@VERSION_PATCH@
 # for a project that appears at the top of each page and should give viewer a
 # quick idea about the purpose of the project. Keep the description short.
 
-PROJECT_BRIEF          = 
+PROJECT_BRIEF          =
```

Comment:
> Quite coincidentally, the last 2 _big_ changes to it were fc850ed99fc3e52e16c7d9d1bf71daf92cbe5f76 and 8b7ac65aa35dc99a61e8c48f801961e49f12564a, the former removing trailing spaces and the latter introducing them again. 

That's exactly what I meant with "trouble" :grin:
No, I'm fine either way. But don't be surprised if someone accidentally introduces spaces again :stuck_out_tongue: